lms/djangoapps/courseware/courses.py

```diff
@@ -373,11 +373,12 @@ def get_course_syllabus_section(course, section_key):
     raise KeyError("Invalid about key " + str(section_key))
 
 
-def get_courses(user, domain=None):
-    '''
-    Returns a list of courses available, sorted by course.number
-    '''
-    courses = branding.get_visible_courses()
+def get_courses(user, domain=None, org=None):  # pylint: disable=unused-argument
```

Comment:
The `domain` kwarg is part of a dead code path that [wasn't completely cleaned up](https://github.com/edx/edx-platform/commit/a3211a7405d92d09d59d3e2fe9ef437051324bcb#diff-fddcdf03b768a768d398d80b8798e76dL297). I'll open a separate PR with the cleanup.